resources/cards/models/card_editable.yml

```diff
@@ -0,0 +1,23 @@
+CardEditable:
+  title: CardEditable
+  required:
+  - description
+  - front
+  type: object
+  properties:
+    description:
+      title: Description
+      maxLength: 255
+      type: string
+    front:
+      title: Front
+      type: string
+      format: path
+    back:
+      title: Back
+      type: string
+      format: path
+    size:
```

Comment:
I don't think size is an editable field for cards, the only fields we allow are description, and then both (auto_reorder and reorder_quantity) since both of those are required together